src/views/UserProfile.vue

```diff
@@ -44,6 +46,7 @@
                       chips
                       multiple
                       disabled
+                      id="groups"
```

Comment:
Should probably namespace these ids to prevent them conflicting with other components.

E.G. GScan will likely have an element `#groups`

Perhaps `profile.groups`, `profile-groups`, whatever.

Reply:
Oh, good idea!